src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -213,7 +213,7 @@ internal Binder GetSpeculativeBinder(int position, ExpressionSyntax expression,
                 }
             }
 
-            Binder binder = this.GetEnclosingBinder(position);
+            Binder binder = GetEnclosingBinder(position);
```

Comment:
Please revert style-only changes in the compiler layer.

Reply:
Why?

Reply:
1. In PRs which make functional changes, we want to be able to focus on reviewing your functional changes and not be scrolling through style changes at the same time.
2. The compiler avoids making style-only changes in general as it's considered to be disruptive to the history and can make it more difficult to merge feature branches, e.g. if the base makes a style-only change and a feature branch makes a functional change to the same area.